config/logback.xml

```diff
@@ -0,0 +1,138 @@
+<!--
+ Copyright 2022 The Apache Software Foundation
+
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+ Define some default values that can be overridden by system properties
+-->
+<configuration>
+  <!-- Uncomment this if you would like to expose Logback JMX beans -->
+  <!--jmxConfigurator /-->
+
+  <property name="zookeeper.console.threshold" value="INFO" />
+
+  <property name="zookeeper.log.dir" value="/app/logs" />
+  <property name="zookeeper.log.file" value="/app/logs/zk-app.log" />
+  <property name="zookeeper.log.threshold" value="INFO" />
+  <property name="zookeeper.log.maxfilesize" value="1MB" />
+  <property name="zookeeper.log.maxbackupindex" value="20" />
+
+  <!--
+    console
+    Add "console" to root logger if you want to use this
+  -->
+  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{ISO8601} [myid:%X{myid}] - %-5p [%t:%C{1}@%L] - %m%n</pattern>
+    </encoder>
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+      <level>${zookeeper.console.threshold}</level>
+    </filter>
+  </appender>
+
+<appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <file>/app/logs/zk-app.log</file>
+    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+      <!-- daily rollover -->
+      <fileNamePattern>/app/logs/zk-app.log.%d{yyyy-MM-dd}.%i</fileNamePattern>
```

Comment:
did you check if our filebeat can grab the files?

iirc it looks for  *.log  Not sure if *.log.<date>.<increment> wil be working

Reply:
Updated file name formate to `/app/logs/zk-app-%d{yyyy-MM-dd}.log.gz`
It is rollover file.  filebeat should not grab the file, if the file ends with *.log those logs will be duplicated 

for reference, I reviewed the elastic search service
![Screenshot from 2023-03-17 12-52-14](https://user-images.githubusercontent.com/115966674/225840791-e286e581-6f87-44a5-ba66-2357213fb3c5.png)